aten/src/ATen/core/builtin_function.h

```diff
@@ -110,6 +116,8 @@ struct BuiltinOpFunction : public Function {
   std::function<void(Stack&)> callable_;
 
   c10::FunctionSchema schema_;
+
+  std::string doc_string_;
```

Comment:
just a bit curious here, why we added the `doc_string_` inside `BuiltinOpFunction` instead of `Function`?

Reply:
`Function` seems to be purely an interface. If we put a member here, then it either 1) limits how subclasses implement doc_string, maybe they want to generate it on the fly 2) or the member is a waste of memory. Let me know what you think.

Reply:
discussed offline, let's keep doc_string only in builtinOpFunction as not all subclass of Function need this doc_string.